Plugins/BackbonePluginTemplatePlugin/BackbonePluginTemplatePluginGenerator.cs

```diff
@@ -131,6 +140,87 @@ private GlobalJsonModification CreateMsBuildSdk()
             return globalJsonModificationBuilder.Build();
         }
 
+        private Task AddProjectReferences()
+        {
+            var packagesReferences = CreateProjectModification();
+            return _projectManipulator.ManipulateProject(GetProjectFilePath(), packagesReferences);
+        }
+
+        private Task AddTestProjectReferences()
+        {
+            var testProjectModification = CreateTestProjectModification();
+            return _projectManipulator.ManipulateProject(GetTestProjectFilePath(), testProjectModification);
+        }
+
+        private ProjectModification CreateProjectModification()
+        {
+            var projectModificationBuilder = new ProjectModificationBuilder();
+            if (PluginVariant != VariantType.MinimalPlugin)
```

Comment:
Add a newline before this

Reply:
Addressed in: [Add newline before if statement](https://github.com/messerli-informatik-ag/meta-generator/pull/27/commits/3e802d861711db6b5d6636eee779c7eb3b893f5f)